finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -13,6 +13,8 @@ private[netty4] class ServerBridge[In, Out](
     serveTransport: Transport[In, Out] => Unit)
   extends ChannelInitializer[Channel] {
 
+  override def handlerAdded(ctx: ChannelHandlerContext): Unit = {}
```

Comment:
Why we need this? If that's to prevent init to be called maybe we should just rebrand it as a plain `InboundChannelHandler` to be explicit about what we want?